Lib/fontTools/ttLib/__init__.py

```diff
@@ -862,8 +870,8 @@ def _escapechar(c):
 		return hex(byteord(c))[2:]
 
 
-def tagToIdentifier(tag):
-	"""Convert a table tag to a valid (but UGLY) python identifier,
+def nameToIdentifier(name):
+	"""Convert a name to a valid (but UGLY) python identifier,
```

Comment:
this docstring is too verbose, it's a straight copy paste from tagToIdentifier. Let's just point to the other one and mention the differences or what's the purpose of this one. Or simply remove the docstring.

Reply:
For this use case, it is of no importance that the result is a valid Python identifier. `.notdef` might as well translate to `2e_n_o_t_d_e_f`, without the leading underscore.

Reply:
I'll update the docstring as @anthrotype  suggests; @justvanrossum you are correct that the requirements aren't as strict for this case - however since we already have logic that creates identifiers which could be used, I didn't want to introduce another, separate, generator.